__tests__/app.test.js

```diff
@@ -176,4 +177,41 @@ describe('POST /api/reviews/:review_id/comments', () => {
         .expect(400)
     });
 });
+
+    describe('PATCH /api/reviews/:review_id', () => {
+        it('should return an updated review with incremented votes ', () => {
+            const input = {inc_votes: 1}
+            return request(app)
+            .patch('/api/reviews/1')
+            .send(input)
+            .expect(202)
+            .then(({body}) => {
+                expect(body.votes).toBe(2)
+            })
+         });
+         it('should return an updated review with decremented votes ', () => {
+            const input = {inc_votes: -100}
+            return request(app)
+            .patch('/api/reviews/1')
+            .send(input)
+            .expect(202)
+            .then(({body}) => {
+                expect(body.votes).toBe(-99)
+            })
+         });
+         it('should return a 404 with invalid path  ', () => {
```

Comment:
These last two descriptions aren't quite right, you're checking firstly for a valid but non existant article ID, then checking an invalid datatype article ID